docs/api/networksocket/CellularNonIPSocket.md

```diff
@@ -0,0 +1,83 @@
+## Non-IP cellular socket
+
+The CellularNonIPSocket class provides, through standard socket send and recv member functions, the ability to send and receive Non-IP datagrams using Control Plane CIoT EPS optimisation feature of the NB-IoT networks. This feature is implemented in ControlPlane_netif class. [`ControlPlane_netif`](https://os.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_control_plane__netif.html) class.
+
+The constructor takes no parameters. To initialize the socket on a specified NetworkInterface, you must call `open` method, which takes a CellularContext pointer.
+
+[`CellularContext`](https://os.mbed.com/docs/development/mbed-os-api-doxy/_cellular_context_8h.html) is setting up the modem into the Control Plane CIoT EPS optimisation mode of operation if it is requested and cellular network supports it.
+
```

Comment:
add
"Control plane optimisations is a new feature for Cellular IoT (CIoT), where the device can use cellular control channels for data communications, to save power and resources as there is less radio singling needed. 

NOTE! Non-IP usage usually requires integration to cellular operator messaging service. The service supports a web API to send and receive to the non-IP using devices.  "

Reply:
done